CIScripts/BuildStage.ps1

```diff
@@ -9,8 +9,11 @@ $OutputRootDirectory = "output"
 $NothingToBuild = $Env:COMPONENTS_TO_BUILD -eq "None"
 $CopyDisabledArtifacts = Test-Path Env:COPY_DISABLED_ARTIFACTS
 
+$IsReleaseMode = [bool]::Parse($Env:BUILD_IN_RELEASE_MODE)
+$BuildMode = $(if ($IsReleaseMode) { "production" } else { "debug" })
+
 if (-not $NothingToBuild) {
-    & $PSScriptRoot\Build.ps1
+    & $PSScriptRoot\Build.ps1 -BuildMode $BuildMode
```

Comment:
Thought: it's not entirely clear from $BuildMode variable name what it means. This seems to "leaks" from internal implementation details.

Reply:
I tried to hide the implementation details by adding a helper function, but maybe it's not an elegant solution. What do you think?